messages/context.json

```diff
@@ -117,6 +117,8 @@
   "pages.editor.store.settings.title": "pages.editor.store.settings.title",
   "pages.editor.store.settings.error.title": "pages.editor.store.settings.error.title",
   "pages.editor.store.settings.error": "pages.editor.store.settings.error",
+  "pages.editor.store.settings.tabs.general": "pages.editor.store.settings.tabs.general",
+  "pages.editor.store.settings.tabs.advanced": "pages.editor.store.settings.tabs.advanced",
```

Comment:
⛏ We've been trying to keep i18n keys sorted alphabetically (ascending order).

Reply:
Somehow, the lint --fix didn't do the trick 🤷‍♂️